src/anemoi/training/config/training/default.yaml

```diff
@@ -115,3 +115,8 @@ pressure_level_scaler:
   _target_: anemoi.training.data.scaling.ReluPressureLevelScaler
   minimum: 0.2
   slope: 0.001
+
+node_loss_weights:
+  _target_: anemoi.training.losses.nodeweights.GraphNodeAttribute
```

Comment:
Would be great to see this, and the stretched-grid option documented in the docs.